### DISCUSSION TITLE
Issue with sending XCH to external adress

### DISCUSSION THREAD

**Discussion by coucou2003** [Chia-Network/chia-blockchain#17178] [Support]
The 22 December i send 8  XCH to my external wallet 
after few days i get only 5 XCH in my address
what the issue with that transaction
![Capture2](https://github.com/Chia-Network/chia-blockchain/assets/155120224/cdbf3983-8446-4af3-ad1f-3c95e3893923)

**Comment by BrandtH22** [comment]
Hey @coucou2003 , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia)

As for the transaction, from what I am seeing onchain the 8xch transaction never went through and the 5xch transaction was sent to your external xch address (https://www.spacescan.io/address/xch1e2k0rld80qhtmdj9vatru45p22u45camu98u22xlkzps383wrzqqh4rdjq).

What I would do is resync the wallet (gui settings / advanced / resync) and upon resyncing send just the 3 xch transaction with a reasonable fee to the external address.

Note - this can occur when multiple different transactions are attempted before previous once are confirmed onchain, one transaction will go through and the wallet seeing the coin spent can be mislead into thinking a different transaction had gone through. Our team is working on better tooling to review the actual coins in the wallet and improve the reattempting of transactions.

Let us know if any questions arise 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
